### PR TITLE
[HelmDeployV0] Fixed tests for debug mode

### PR DESCRIPTION
**Pull request by egor-bryzgalov**
**Task name**: HelmDeployV0

**Description**: The task adds the `--debug` flag to commands when a pipeline running HelmDeployV0 tests runs in debug mode. This caused some pipelines to have failed. [An example of the failed pipeline is here.](https://dev.azure.com/mseng/PipelineTools/_build/results?buildId=14129905&view=results)
This PR fixes tests for debug pipelines.

**Documentation changes required:** No

**Added unit tests:** Fixed existing

**Attached related issue:** No

**Checklist**:
- [x] Checked that applied changes work as expected

### CODE REVIEW

File: Tasks/HelmDeployV0/Tests/L0.ts [hunk 1]
```diff
@@ -217,6 +226,7 @@ describe("HelmDeployV0 Suite", function () {
         process.env[shared.isHelmV3] = "true";
 
         tr.run();
+        console.log(tr.stdout);
```
Comment (anatolybolshakov) [inline Tasks/HelmDeployV0/Tests/L0.ts:229]: Redundant changes

File: Tasks/HelmDeployV0/Tests/TestSetup.ts [hunk 2]
```diff
@@ -97,6 +98,10 @@ const a: ma.TaskLibAnswers = <ma.TaskLibAnswers>{
 if (process.env[shared.TestEnvVars.command] === shared.Commands.install) {
     let helmInstallCommand = "helm install";
 
+    if (debug) {
+        helmInstallCommand = helmInstallCommand.concat(` --debug`);
```
Comment (anatolybolshakov) [inline Tasks/HelmDeployV0/Tests/TestSetup.ts:102]: Could it make sense to put it to some method - to avoid duplication?

File: Tasks/HelmDeployV0/Tests/L0.ts (hunk 1 above)
Comment (egor-bryzgalov) [reply Tasks/HelmDeployV0/Tests/L0.ts]: Removed them since they were temporary.

**Review by anatolybolshakov — APPROVED** [review]

File: Tasks/HelmDeployV0/Tests/TestSetup.ts (hunk 2 above)
Comment (egor-bryzgalov) [reply Tasks/HelmDeployV0/Tests/TestSetup.ts]: Thanks. Applied change.

**Review by EzzhevNikita — APPROVED** [review]

**Comment by ds-ms** [comment]
@shigupt202 could you take a look at this PR and take it forward

**Review by anatolybolshakov — APPROVED** [review]
LGTM, thanks!
Please also update current description of addDebugFlag

**Review by alexander-smolyakov — APPROVED** [review]

**Comment by ds-ms** [comment]
Thanks @egor-bryzgalov for fixing this edge case in tests. 😃

**Comment by egor-bryzgalov** [comment]
@ds-ms sometimes it may be needed to run repository CI pipelines or the pipeline creating a release branch and so on in debug mode to investigate an issue. If HelmDeployV0 tests run within that pipeline it would fail and would make someone confused. I faced this issue while working on the pipeline mentioned in the description.
